### ISSUE TITLE
[CSS] Delete display:flex from all pb-autocomplete elements

### ISSUE THREAD

**Issue by plutonik-a** [eeditiones/tei-publisher-components#44]
`pb-automplete` simply consists of an input and a label.
Thus a rule `display: flex;` is set globally, is unnecessary and causes layout problems when adding more inputs into a form, or in mixing with other flexed elements. 

I guess, the initial reason for this rule was to insert elements like a select element into the component, so that the select and input are displayed inline. 
(-> Example "Combine with other form fields whose value is passed on" https://unpkg.com/@teipublisher/pb-components@1.2.0/dist/api.html#pb-autocomplete.1)

<img width="259" alt="Screenshot 2020-09-14 at 20 13 09" src="https://user-images.githubusercontent.com/1157669/93122443-c35f9480-f6c6-11ea-90fd-cb7193459add.png">

This rule should be set by either placing a customizable variable into the webcomponent, or just left blank.

**Comment by tuurma** [comment]
Indeed, the default behaviour of nested components (eg when combining pb-autocomplete with pb-select) should be aligned with the parent, like in the *Combine with other form fields...* example.

Would you care to create a PR for develop branch that assures this default behaviour is retained while not causing side-effects?

**Comment by JoernT** [comment]
layout should certainly as basic as possible and such specific settings should be avoided. Further i found that often it's a good idea to just use 'inherit' on the :host matcher in a component to overcome such issues.
